### PR TITLE
TH-459 | Allow excluding contract zones from import

### PR DESCRIPTION
**Pull request by tuomas777**

### CODE REVIEW

**Comment by tuomas777** [comment]
It turned out that contract zone IDs might change in the future, so we need to use names instead of IDs when excluding them.

File: areas/importer/helsinki.py [hunk 1]
```diff
@@ -22,13 +23,22 @@ def import_contract_zones(self):
         logger.info("Helsinki contract zone import done!")
 
     def _fetch_contract_zones(self):
+        contract_zone_filter_str = (
+            ' AND "nimi" NOT IN ({})'.format(
+                ",".join(("'{}'".format(c) for c in EXCLUDED_CONTRACT_ZONES))
```
Comment (igordavydsson) [inline areas/importer/helsinki.py:28]: absolutely minimal nitpick: would `for cz in EXCLUDED_CONTRACT_ZONES` make more sense than just `c`? 😅

Comment (tuomas777) [reply areas/importer/helsinki.py]: Yes it would! Does it matter at all, probably not! But changed anyway! 😄

Comment (igordavydsson) [reply areas/importer/helsinki.py]: yeah, not that it matters, but all nitpicks matter 🙌 😅

**Pull request merged**